<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="picture">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>740</width>
      <height>340</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Picture</string>
    </property>
    <widget class="QLabel" name="InputImgBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Input Image</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="OutputImgBox">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>OutPut Image</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Log">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>520</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="title">
     <string>Log</string>
    </property>
    <widget class="QTextBrowser" name="LogBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>500</width>
       <height>150</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>569</x>
      <y>370</y>
      <width>201</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
     </font>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QPushButton" name="select_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>160</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>选择图片</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UpLoad">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>160</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>上传</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BatchButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>160</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>批处理</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CleanLog">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>160</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清空日志</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
